spec/support/fixtures/grant_code_exchange_response.json

```diff
@@ -0,0 +1,8 @@
+{
+    "access_token": "fake-access-token",
+    "expires_in": 28799,
```

Comment:
It helps to know whether the OAuth token needs to be refreshed. A "hack" is to not store it and just try to exchange if you get an OAuth error back from the Platform API -- but that can be problematic, too since it doubles the amount of requests you're making and requires some database writing to do it all in one request/response.

Reply:
saving `access_token_expires_at` timestamp via 8e48c79